sklearn/neighbors/_unsupervised.py

```diff
@@ -5,22 +5,23 @@
 from ._base import UnsupervisedMixin
 
 
-class NearestNeighbors(NeighborsBase, KNeighborsMixin,
-                       RadiusNeighborsMixin, UnsupervisedMixin):
+class NearestNeighbors(
+    NeighborsBase, KNeighborsMixin, RadiusNeighborsMixin, UnsupervisedMixin
+):
```

Comment:
Please don't make unrelated code style changes, and they create merge conflicts for other PRs and also make using `git blame` more difficult.

Reply:
Understood and reverted, thanks.